lib/api/bucketPutPolicy.js

```diff
@@ -70,6 +69,16 @@ function bucketPutPolicy(authInfo, request, log, callback) {
                 return next(null, bucketPolicy);
             });
         },
+        (bucketPolicy, next) => {
```

Comment:
Do we need an extra step in the waterflow? I think we could just add the `validatePolicyConditions` block right after the `validatePolicyResource` one above.

Reply:
More like [this](https://github.com/scality/cloudserver/pull/5501/files#diff-f9ed4547634b44b1019792bc2c7bdd3fd5c512d9e0f6ed3ba3c67b62b8745efaR69)?